CirclesUBI.PathfinderUpdater.IndexerDb/Block.cs

```diff
@@ -4,7 +4,7 @@ namespace CirclesUBI.PathfinderUpdater;
 
 public static class Block
 {
-    public static async Task<long> FindByTransactionHash(string connectionString, string transactionHash, Queries queries)
+    public static async Task<long> FindLatestBlockNumber(string connectionString, Queries queries)
```

Comment:
just curious how the function name and signature changed but nothing else in the function?

Reply:
I think i understand how it worked before, and see that now you want to go over every block?

Reply:
Good catch. That's basically dead code that was once needed to do partial updates to the pathfinder.